vue.config.js

```diff
@@ -56,6 +56,50 @@ module.exports = {
             to: 'public'
           }
         ],
+        fileAssociations: [
+          {
+            name: "SQLite File",
+            ext: ["db", "sqlite", "sqlite3"],
+            role: "Editor"
+          }
+        ],
+        protocols: [
```

Comment:
Looking at the docs, it seems like `protocols` is meant to be a part of a fileAssociation?

I might be wrong (the docs are confusing). Have you tested this as working?

Reply:
I think you have it right, looking at this commit - https://github.com/electron-userland/electron-builder/blob/28cb86bdcb6dd0b10e75a69ccd34ece6cca1d204/packages/app-builder-lib/src/options/PlatformSpecificBuildOptions.ts

Reply:
yes this was working for me! I built it and tested locally, as well as manually inspected the generated `Info.plist`.

Actually, I was very confused at first as well, I think the docs are misformatted. I had to read the [source](https://github.com/electron-userland/electron-builder/blob/28cb86bdcb6dd0b10e75a69ccd34ece6cca1d204/packages/app-builder-lib/src/options/PlatformSpecificBuildOptions.ts#L74-L81) to confirm.

Reply:
we are not the only ones confused 😂: https://github.com/electron-userland/electron-builder/issues/5286

Reply:
lol yes.